code/modules/tgui/states.dm

```diff
@@ -16,14 +16,20 @@
  **/
 /datum/proc/ui_status(mob/user, datum/ui_state/state)
 	var/src_object = ui_host()
+	. = UI_CLOSE
```

Comment:
I'm guessing the purpose of this is to allow `admin_state` when dead? Fair enough, that was an oversight when I added the ghost interaction.